JustSaying/AwsTools/MessageHandling/SqsQueueByNameBase.cs

```diff
@@ -90,12 +92,13 @@ public virtual async Task<bool> CreateAsync(SqsBasicConfiguration queueConfig, i
                 // If we're on a delete timeout, throw after 2 attempts.
                 if (attempt >= 2)
                 {
-                    _log.LogError(0, ex, "Error trying to create queue '{QueueName}'. Maximum retries exceeded for delay.", QueueName);
+                    _log.LogError(0, ex, "Error trying to create queue '{QueueName}'. Maximum retries exceeded for delay {Delay}.",
```

Comment:
Maybe we should include what the maximum number of retries is?

Reply:
That literal `2` of course means that there will be 3 attempts, numbered 0,1,2

Reply:
And it needs to be moved a few lines earlier if it is to do anything.